internal/controller/postgrescluster/controller.go

```diff
@@ -504,9 +507,22 @@ func (r *Reconciler) SetupWithManager(mgr manager.Manager) error {
 		}
 	}
 
+	// K8SPG-712: Allow overriding default configurations
+	configMapPredicate := builder.WithPredicates(predicate.Funcs{
```

Comment:
Maybe we can cover this in our controller tests if it not major change for the controller_test.go.

Reply:
https://github.com/percona/percona-postgresql-operator/pull/1108/commits/b3211a04fb5d9cd1c0517df80885c8c372245d48